samples/stock/indicators/ema/demo.js

```diff
@@ -1,5 +1,4 @@
-
-$.getJSON('https://www.highcharts.com/samples/data/aapl-ohlc.json', function (data) {
+$.getJSON('https://www.highcharts.com/samples/data/jsonp.php?filename=aapl-ohlc.json&callback=?', function (data) {
```

Comment:
We stopped using the jsonp.php endpoints, instead we opened up for CORS directly.